machine/info.go

```diff
@@ -173,20 +174,11 @@ func Info(sysFs sysfs.SysFs, fsInfo fs.FsInfo, inHostNamespace bool) (*info.Mach
 }
 
 func ContainerOsVersion() string {
-	container_os := "Unknown"
-	os_release, err := ioutil.ReadFile("/etc/os-release")
-	if err == nil {
-		// We might be running in a busybox or some hand-crafted image.
-		// It's useful to know why cadvisor didn't come up.
-		for _, line := range strings.Split(string(os_release), "\n") {
-			parsed := strings.Split(line, "\"")
-			if len(parsed) == 3 && parsed[0] == "PRETTY_NAME=" {
-				container_os = parsed[1]
-				break
-			}
-		}
+	os, err := operatingsystem.GetOperatingSystem()
+	if err != nil {
+		os = "Unknown"
```

Comment:
do we want to log the error V(4)?  Just in-case we get support questions about unknown OS.

Reply:
We didn't before so I just maintained what the old code did before i.e. returning "Unknown" if it couldn't be parsed.

My opinion is that we shouldn't log.  The "Unknown" in the `kubectl get nodes` output is evidence enough that the parsing failed.

Reply:
ok, sgtm.  Thanks!